info/available-rules.md

```diff
@@ -1,127 +1,128 @@
-|    6  |     6.1.9  |  WRONG_NAME_OF_VARIABLE_INSIDE_ACCESSOR  |  Check: used the name of a variable in the custom getter or setter                                                                                                             |   no  |  no |
-|    6  |    6.1.10  |  TRIVIAL_ACCESSORS_ARE_NOT_RECOMMENDED   |  Check: if there are any trivial getters or setters <br> Fix: Delete trivial getter or setter                                                                                  |  yes  |  no  |  - |
-|    6  |     6.1.8  |  CUSTOM_GETTERS_SETTERS                  |  Check: Inspection that checks that no custom getters and setters are used for properties                                                                                      |  no  |  no  |  - |
-|    6  |    6.1.11  |  COMPACT_OBJECT_INITIALIZATION           |  Checks if class instantiation can be wrapped in `apply` for better readability  |    |  no  |  |
-|    6  |    6.1.12  |  INLINE_CLASS_CAN_BE_USED                |  Check: warns if class can be transferred to the inline class.                                                                                                                 | no | no | yes |
-|    6  |     6.2.2  |  EXTENSION_FUNCTION_SAME_SIGNATURE       |  Checks if extension function has the same signature as another extension function and their classes are related                                                               |  no  |  no  |  + |
-|    6  |     6.4.1  |  AVOID_USING_UTILITY_CLASS               | Checks if there is class/object that can be replace with extension function                                                                                                  | no  | no | - |
-|    6  |     6.4.2  |  OBJECT_IS_PREFERRED                     | Checks: if class is stateless it is preferred to use `object`                                                                                                                | yes | no | + |
-|    6  |     6.2.3  | EXTENSION_FUNCTION_WITH_CLASS            | Check: if file contains class, then it can not have extension functions for the same class                                                                                                     | no | no | - |
-|    6  |     6.5.1  |  RUN_IN_SCRIPT                           | Checks : if kts script contains other functions except run code                                                                                                              | yes | no | - |
\ No newline at end of file
+| Chap  |  Standard  | Rule name                                | Description                                                                                                                       | Fix | Config | FixMe                                                                                                                                                                                                                         |
```

Comment:
Why full file is changed? Some changes in end lines?

Reply:
Probably `\n` was replaced with `\r\n`

Reply:
Please configure your git to use only `LF` in commits [How to change line-ending settings](https://stackoverflow.com/questions/10418975/how-to-change-line-ending-settings)